src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -1,45 +1,96 @@
 <div
   class="sky-form-group sky-input-group sky-phone-field"
+  [@initalLoad]
```

Comment:
I like this technique. Should we rename it to something like `blockAnimationOnLoad`?

Reply:
Done